lib/matplotlib/animation.py

```diff
@@ -1251,10 +1251,10 @@ def _setup_blit(self):
         self._blit_cache = dict()
         self._drawn_artists = []
         self._resize_id = self._fig.canvas.mpl_connect('resize_event',
-                                                       self._handle_resize)
+                                                       self._on_resize)
         self._post_draw(None, self._blit)
 
-    def _handle_resize(self, *args):
+    def _on_resize(self, *args):
```

Comment:
do you want to fix the signature as well?

Reply:
Sure.